www/front_src/src/components/navigation/index.js

```diff
@@ -268,11 +241,17 @@ class NavigationComponent extends Component {
                                     if (levelFourProps.label) {
                                       return (
                                         <li
-                                          class={"collapsed-level-item" + (this.isActive(pageId, levelFourUrl) ? " active" : "")}
+                                          onClick={() => {
+                                            this.state.active ?
+                                              this.activeCurrentLevel(levelOneKey, levelTwoKey)
```

Comment:
bad coding style. Should be : 
expression
  ? result1
  : result2

Reply:
done

Reply:
Did not see this, indeed. But there is no need for the ternary and active should be deconstructed.